src/vs/workbench/services/configurationResolver/common/configurationResolverSchema.ts

```diff
@@ -102,7 +102,7 @@ export const inputsSchema: IJSONSchema = {
 								description: nls.localize('JsonSchema.input.command.command', "The command to execute for this input variable.")
 							},
 							args: {
-								type: 'object',
+								type: 'array',
```

Comment:
Changing args to be an array breaks all old launch configs not using arrays. So we will have to support both: args can be an object or an array of objects.

Reply:
Updated to allow object, array, and also string. This provides backward compatibility and makes it totally consistent with keybindings.